charts/keycloak/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: keycloak
-version: 10.1.0
+version: 10.1.1
```

Comment:
please increase version to be one bugfix above the current one

Reply:
Done. Thank you.

Reply:
Sorry for the long wait. The version increased again in the meantime... Please rebase the branch and increment it again